code/modules/clothing/ears/ears.dm

```diff
@@ -1,50 +1,50 @@
-
-//Ears: currently only used for headsets and earmuffs

```

Comment:
This showed up when merge conflicts were resolved; I have no idea why; nothing was changed here.